src/documentation/0053-node-difference-dev-prod/index.md

```diff
@@ -33,28 +33,28 @@ Setting the environment to `production` generally ensures that
 
 For example Pug, the templating library used by Express, compiles in debug mode if `NODE_ENV` is not set to `production`. Express views are compiled in every request in development mode, while in production they are cached. There are many more examples.
 
-Express provides configuration hooks specific to the environment, which are automatically called based on the NODE_ENV variable value:
+You can use conditional statements to execute code in different environments:
 
 ```js
-app.configure('development', () => {
+if (app.get('env') === "development") {
```

Comment:
Perhaps we can use `process.env.NODE_ENV` instead of `app.get('env')` since the last one is specific from an express instance and also `Express` is being removed from the example description.